usr/sbin/p11sak/p11sak.c

```diff
@@ -2230,7 +2230,6 @@ static CK_RV finalize_destroy_object(char *label, CK_SESSION_HANDLE *session,
             goto done;
         }
         *boolDestroyFlag = CK_TRUE;
-        printf("DONE - Destroy Object with Label: %s\n", label);
```

Comment:
Why don't you combine patch 1 and 2 into just one patch? This seems to change a lone also touched in first patch.